modules/admin_manual/pages/configuration/server/occ_commands/app_commands/_full_text_search_commands.adoc

```diff
@@ -68,9 +69,47 @@ Indexing user admin
 Indexing user testuser
 ----
 
+== Fill a Secondary Index
+
+Fill a secondary index based on the indexed data we have. Files not matching the "indexed" status will be ignored. This is intended to be used in index data migrations, so the connector for this secondary index should have been configured as "write connector". See the xref:configuration/general_topics/search.adoc[Full Text Search] documentation for more details. Note that the values of the `connector_name` are hardcoded and have to be written as described.
+
+[source,bash,subs="attributes+"]
+----
+{occ-command-example-prefix} search:index:fillSecondary [options] [--] <connector_name> <user_id>...
+----
+
+=== Arguments
+
+[width="100%",cols="20%,100%"]
+|===
+| connector_name
+| The name of the connector. `Legacy` (default) or `RelevanceV2`.
+
+| `<user_id>` 
+| Provide a userId. This argument is required.
+|===
+
+=== Options
+
+[width="100%",cols="20%,100%"]
+|===
+| `-q` `--quiet` 
+| Do not output any message.
+
+| `-f` `--force` 
+| Use this option to fill the secondary index without further questions.
+
+| `--startOver`
+| Start indexing from the beginning, not from a previous savepoint.
+
+| `--chunkSize=CHUNKSIZE`
+| The savepoint will be updated after processing this number of files. [default: "100"]
+|===
+
 == Rebuild the Full Text Search Index
 
-Rebuild the search index for a given user. +
+Rebuild the indexes for a given User. All the indexes associated with the configured connectors will be rebuilt. This won't apply any change to the configuration of the index if it's already setup but it will setup any index that hasn't been setup yet. Check "search:index:reset" to reset all the indexes associated to the configured connectors
```

Comment:
```suggestion
Rebuild the indexes for a given user. All the indexes associated with the configured connectors will be rebuilt. This won't apply any change to the configuration of the index if it's already set up but it will set up any index that hasn't been set up yet. Check "search:index:reset" to reset all the indexes associated to the configured connectors.
```